arithmetization/src/main/java/net/consensys/linea/zktracer/module/hub/fragment/imc/ImcFragment.java

```diff
@@ -20,18 +20,20 @@
 
 import net.consensys.linea.zktracer.module.hub.Hub;
 import net.consensys.linea.zktracer.module.hub.Trace;
+import net.consensys.linea.zktracer.module.hub.defer.ContextReEntryDefer;
 import net.consensys.linea.zktracer.module.hub.fragment.TraceFragment;
 import net.consensys.linea.zktracer.module.hub.fragment.TraceSubFragment;
 import net.consensys.linea.zktracer.module.hub.fragment.imc.exp.ExpCall;
 import net.consensys.linea.zktracer.module.hub.fragment.imc.mmu.MmuCall;
 import net.consensys.linea.zktracer.module.hub.fragment.imc.oob.OobCall;
+import net.consensys.linea.zktracer.runtime.callstack.CallFrame;
 import net.consensys.linea.zktracer.types.TransactionProcessingMetadata;
 
 /**
  * IMCFragments embed data required for Inter-Module Communication, i.e. data that are required to
  * correctly trigger other modules from the Hub.
  */
-public class ImcFragment implements TraceFragment {
+public class ImcFragment implements TraceFragment, ContextReEntryDefer {
```

Comment:
We need `ContextReEntryDefer` in order to be able to fill in the following fields:
```rust
CCSR = CHILD_CONTEXT_SELF_REVERTS
CCRS = CHILD_CONTEXT_REVERT_STAMP
```
These values are used by CALL and CREATE to set future undoing row `DOM`/`SUB` stamps related to failed internal transactions.